### PR TITLE
moved individual css to style.css

### PR DESCRIPTION
**Pull request by airenzp**

### CODE REVIEW

**Review by jonandernovella — APPROVED** [review]

**Review by HenrikeW — CHANGES_REQUESTED** [review]
These changes introduce problems with padding and margin in the individual form because the built-in styles of the Material UI elements overwrite the plain CSS styles. 
Earlier, in similar cases, we solved this with inline styles which might work here as well, even if it's bad practice. Maybe there is a better way I can't think of right now?
Anyway, this should be fixed before we merge this.

**Comment by airenzp** [comment]
Made static css to work with MaterialUI by using StylesProvider injectFirst property at the main page!

File: frontend/src/style.css [hunk 1]
```diff
@@ -445,3 +394,155 @@ list-item {
...
+    padding: 0 20px;
+  }
+
+  .formPane {
+    border-right: 1px solid lightgrey;
+  }
+
+  .formPane {
+    border-right: 1px solid lightgrey;
+  }
```
Comment (HenrikeW) [inline frontend/src/style.css:525]: This is a duplicate to the rows above.

File: frontend/src/static_pages.tsx [hunk 2]
```diff
@@ -202,20 +202,20 @@ export function Kontakt() {
 
 export function Footer() {
   return (
-    <div className="footer">
+    <Paper className="footer">
```
Comment (HenrikeW) [inline frontend/src/static_pages.tsx]: I wonder why you made the footer a `Paper`. Was there a specific reason? I think, now it doesn't look like a real footer any more - it doesn't stick to the bottom of the page and has space around it on the left and right. 
Not sure if we should focus on this now but I think, at least at some point, it should be improved.

File: frontend/src/main.tsx [hunk 3]
```diff
@@ -9,21 +9,24 @@ import { WithDataContext } from "@app/data_context";
...
-        <WithUserContext>
-          <WithBreedingContext>
-            <WithMessageContext>
-              <Navigation />
-            </WithMessageContext>
-          </WithBreedingContext>
-        </WithUserContext>
-      </WithDataContext>
-    </BrowserRouter>
+    <StylesProvider injectFirst>
```
Comment (HenrikeW) [inline frontend/src/main.tsx:16]: So cool that you could solve it this way! :100:

File: frontend/src/individual_form.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import { InputAdornment, makeStyles, TextField } from "@material-ui/core";
+import { InputAdornment, StylesProvider, TextField } from "@material-ui/core";
```
Comment (HenrikeW) [inline frontend/src/individual_form.tsx]: VSCode marks the `StylesProvider` as unused in this file. Are you sure the import is needed?

**Review by HenrikeW — APPROVED** [review]
Some minor things that you could change/fix if you want to, but nothing severe. Looks good now, I think!

File: frontend/src/style.css (hunk 1 above)
Comment (airenzp) [reply frontend/src/style.css:525]: Thanks, I think I might have forgotten to run the linter in my pc, that detects this duplications, weird that did not jumped on the github linter

File: frontend/src/static_pages.tsx (hunk 2 above)
Comment (airenzp) [reply frontend/src/static_pages.tsx]: the paper component just made round borders, basically. I moved the footer up, but maybe was better as before, I will leave it more at the bottom. The spaces where there before. We might do some redesign later.

**Comment by airenzp** [comment]
@HenrikeW Now the footer is kept at the bottom of the page and looking like a footer, hopefully. I also added the user name at the top of the page after login, so the user can see a change after it gets logged in. And the resize of the window when clicking the menu is gone!

**Pull request merged**